### DISCUSSION TITLE
Missing font family Courier?

### DISCUSSION THREAD

**Discussion by nirvdrum** [FreeCAD/FreeCAD#16430] [Discussions that will turn in to issues but need refining]
When I start FreeCAD 1.0 RC1 I see the following in the "Report view":

```
23:21:45  (qt.qpa.fonts) Populating font family aliases took 56 ms. Replace uses of missing font family "Courier" with one that exists to avoid this cost. 
```

I've noticed throughout the 0.22-dev cycle, but didn't think much of it because I was using unstable builds. I don't think it's actually doing any harm, but I wanted to check if it should be happening at all. Besides switching themes I haven't adjusted any of the UI settings and haven't done anything with fonts.

This is on macOS 14.6.1. I'm entirely open to this being an environment issue on my machine, but wanted to bring it up in case it was a problem with the OS defaults.

**Comment by JohnOCFII** [comment]
Known issue. https://github.com/FreeCAD/FreeCAD/issues/12199

It appears to have something to do with the underlying QT libraries and how it looks for fonts on macOS.

**Reply by nirvdrum** [reply]
Thank you. I'm sorry for the dupe. I searched both the issues and the discussion board for the exact error message and just the word "Courier". GitHub didn't show any related issues, opened or closed. I'll subscribe to that #12199.
